Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -858,7 +858,7 @@ class BrowserViewController: UIViewController {
             return
         }
 
-        if let helper = tab.getContentScript(name: ContextMenuHelper.name()) as? ContextMenuHelper {
+        if tab === tabManager.selectedTab, let helper = tab.getContentScript(name: ContextMenuHelper.name()) as? ContextMenuHelper {
```

Comment:
For added safety, I am going back to just checking on the selected tab. I don't want tabs that are unselected and about to be destroyed getting KVO messages and installing a handler.

Reply:
question... does the `layer.bounds` KVO fire when the selected tab changes?

Reply:
just worried that it is possible to end up in a scenario where we *never* install the handler (although that's better than crashing)

Reply:
Agreed, now that I have the code more robust, I am reverting this to install to any tab so we don't miss one.